helm-chart/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: emx2
-version: 0.0.0
+version: 0.0.1
```

Comment:
I think this happens automatigally; number equal to version number automatic if I understand correctly